app/controllers/comments_controller.rb

```diff
@@ -1,2 +1,17 @@
 class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
```

Comment:
Kindly create the `new` method that initiates a new comment as required by rails conventions for controllers.